<launch>
    <arg name="veh" doc="Name of vehicle. ex: megaman"/>
    <arg name="param_file_name" default="universal" doc="Specify a param file. ex:megaman." />

    <arg name="pkg_name" value="line_detector"/>
    <arg name="node_name" default="line_detector_node"/>

    <arg name="verbose" default="false" />

    <group ns="$(arg veh)">
        <node  name="line_detector_node" pkg="$(arg pkg_name)" type="$(arg node_name).py" output="screen" required="true">
            <rosparam command="load" file="$(find line_detector)/config/$(arg node_name)/$(arg param_file_name).yaml"/>
            <param name="verbose" value="$(arg verbose)"/>
        </node>
    </group>

    <!-- Publications -->

    <!-- Subscriptions -->

</launch>
